ngraph/python/src/pyngraph/node.cpp

```diff
@@ -77,6 +77,8 @@ void regclass_pyngraph_Node(py::module m)
     node.def("get_output_partial_shape", &ngraph::Node::get_output_partial_shape);
     node.def("get_type_name", &ngraph::Node::get_type_name);
     node.def("get_unique_name", &ngraph::Node::get_name);
+    node.def("get_friendly_name", &ngraph::Node::get_friendly_name);
+    node.def("set_friendly_name", &ngraph::Node::set_friendly_name);
 
     node.def_property("name", &ngraph::Node::get_friendly_name, &ngraph::Node::set_friendly_name);
```

Comment:
```suggestion
    node.def_property("friendly_name", &ngraph::Node::get_friendly_name, &ngraph::Node::set_friendly_name);
    node. def_property_readonly("name", &ngraph::Node::get_name);
```

Reply:
After this change it is required to go through all tests that have `name=...` as a Node's parameter. Solutions: 

1. Remove it and leave at as is (since there are no checks in tests for friendly_name) and eventually replace it with a call to set_friendly_name after that. 
2. Modify wrapper to allow friendly_name as a param.

If it is possible I would go with a second one. @postrational what do you think?